.style-and-markdown-build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+ret=0
+
```

Comment:
Not strictly necessary but maybe add a
```suggestion
cd "$(dirname $(realpath -s $0))"
```
That makes it work when it's run from different locations.